pkg/ranger/content.go

```diff
@@ -142,7 +146,7 @@ func ServeContent(w http.ResponseWriter, r *http.Request, name string,
 	w.WriteHeader(code)
 
 	if r.Method != http.MethodHead {
-		r := sendContent()
+		r, _ := sendContent()
```

Comment:
@jtolds I really regret I am doing this, so I'd like to hear your advice. `ServeContent` does not return error and on other places where an error occurs, it is just ignored. Also, I as far as I can see this functions is only used in examples. Thus, I am not sure if we should improve the function to handle errors or we can leave it like this.

Reply:
yeah as soon as w.WriteHeader happens, we lose the ability to communicate with the client that an error happened. we may have already sent a 200. i assume the lack of logging with io.CopyN errors below is due to it being overly chatty from clients disappearing unexpectedly and so on.

i do think you need to check the error, because we are calling r.Close which might panic if r is nil, which is likely if err != nil